src/Servers/Kestrel/Core/src/Internal/Http/HttpProtocol.FeatureCollection.cs

```diff
@@ -342,4 +342,12 @@ Task IHttpResponseBodyFeature.CompleteAsync()
     {
         return CompleteAsync();
     }
+
+#pragma warning disable CA2252 // WebTransport is a preview feature. Suppress this warning
+    public bool IsWebTransportRequest { get; set; }
+    public virtual ValueTask<IWebTransportSession> AcceptAsync(CancellationToken token)
```

Comment:
The other interface members are all explicitly implemented, should these also be implemented explicitly?

Reply:
I can do the first one but, as I then need to override the method in Http3Stream, I must mark it as virtual here, which is not supported if I explicitly implement the method from the interface. Therefore, for consistency and as recommended by Chris, I made both implicit.

Reply:
Resolving as it is a duplicate of another comment. Thank you for reviewing though.